manuscript/template.tex

```diff
@@ -0,0 +1,672 @@
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+\section{Modelling Language and Methodology}
+The languages considered were SysML V1 \cite{sys_ml}, SysML V2 \cite{sys_ml2}, PlantUML \cite{plantuml}, and the Mach 30 modelling language (M30ML) \cite{mach30_git} with the mission architecture team conducting a trade study to determine the most suitable.  The down selection criteria that the SeaLion CubeSat mission architecture team has taken into consideration is shown in table \ref{tab:language}.
+
+\begin{table}[H] 
+	\caption{Modeling Language Downselect}
+	\label{tab:language}
+	\newcolumntype{C}{>{\centering\arraybackslash}X}
+	\begin{tabularx}{\textwidth}{CCCCC}
+	\toprule
+	\textbf{Criteria}  & \textbf{SysML v1}  & \textbf{SysML v2}  & \textbf{PlantUML}  & \textbf{M30ML}\\
+	\midrule
+	Extensible ontology language                       & X        & X        & X        & X     \\ \hline
+	Supports both textual \& graphical view generation &          & X        &          & X     \\ \hline
+	Lightweight textual syntax                         &          & X        & X        & X     \\ \hline
+	Relatively minimal overhead with modern doctools   &          &          & X        & X     \\ \hline
+	Supports execution semantics                       &          & X        &          &       \\
+	\bottomrule
+	\end{tabularx}
+\end{table}
+
+M30ML was chosen for its lightweight human and machine-readable textual syntax, file-based model interchange support (for persisting models directly on the local filesystem), ability to generate both textual and graphical views, and relatively minimal overhead with modern doctools \cite{mach30_git}.  The lightweight textual syntax and minimal over head is especially important for for a team that had very minimal experience working with such tools.  Other candidate modelling languages lacked in many regards compared to M30ML in these criteria and thus, M30ML was selected.  SysML v2 had a good number of characteristics that M30ML had, however, the lack of minimal overhead with modern doctools prevented its adoption.  
+
```

Comment:
Response to https://github.com/ODU-CGA-CubeSat/sealion-mission-architecture/pull/255#issuecomment-1558133218 and https://github.com/ODU-CGA-CubeSat/sealion-mission-architecture/pull/255#issuecomment-1558134566 should probably be addressed more explicitly here, with verbiage s.a., "At the time of writing, current state of the art MBSE languages (s.a., SysML v2) have not prescribed an approach for file-based model interchange for persisting models on the local filesystem".